runtime/compiler/runtime/Runtime.cpp

```diff
@@ -1533,6 +1535,8 @@ void initializeCodeRuntimeHelperTable(J9JITConfig *jitConfig, char isSMP)
    SET(TR_ARM64interpreterDoubleStaticGlue,       (void *) _interpreterDoubleStaticGlue,     TR_Helper);
    SET(TR_ARM64interpreterSyncDoubleStaticGlue,   (void *) _interpreterSyncDoubleStaticGlue, TR_Helper);
    SET(TR_ARM64nativeStaticHelper,                (void *) _nativeStaticHelper,              TR_Helper);
+   SET(TR_ARM64interfaceCompeteSlot2,             (void *) _interfaceCompeteSlot2,           TR_Helper);
+   SET(TR_ARM64interfaceSlotsUnavailable,         (void *) _interfaceSlotsUnavailable,       TR_Helper);
```

Comment:
Are these two "helpers" required?  They're not used and likely a second-generation leftover from Power.  If we decide to employ a PIC solution on AArch64 then we can design and implement what we need then.

Reply:
Removed these entries.